reference.md

```diff
@@ -0,0 +1,1542 @@
+          path to a requirements.txt file
+        - python-packages:
+          (list)
+          A list of dependencies to get from PyPi
+
+
+    $ snapcraft help python3
+    The python3 plugin can be used for python 3 based parts.
+
+    The python3 plugin can be used for python 3 projects where you would
+    want to do:
+
+        - import python modules with a requirements.txt
+        - build a python project that has a setup.py
+        - install sources straight from pip
+
+    This plugin uses the common plugin keywords as well as those for "sources".
+    For more information check the 'plugins' topic for the former and the
+    'sources' topic for the latter.
+
+    Additionally, this plugin uses the following plugin-specific keywords:
+
+        - requirements:
+          (string)
+          path to a requirements.txt file
+        - python-packages:
+          (list)
+          A list of dependencies to get from PyPi
+
+
+    $ snapcraft help qmake
+    The qmake plugin is useful for building qmake-based parts.
+
+    These are projects that are built using .pro files.
+
+    This plugin uses the common plugin keywords as well as those for "sources".
+    For more information check the 'plugins' topic for the former and the
+    'sources' topic for the latter.
+
+    Additionally, this plugin uses the following plugin-specific keywords:
+
+        - options:
+          (list of strings)
+          additional options to pass to the qmake invocation.
+        - qt-version:
+          (enum, 'qt4' or 'qt5')
+          Version of Qt to use with qmake.
+        - project-files:
+          (list of strings)
+          list of .pro files to pass to the qmake invocation.
+
+
+    $ snapcraft help scons
+    The scons plugin is useful for building parts that build with scons.
+
+    These are projects that have a SConstruct that drives the build.
+
+    This plugin uses the common plugin keywords as well as those for "sources".
+    For more information check the 'plugins' topic for the former and the
+    'sources' topic for the latter.
+
+    Additionally, this plugin uses the following plugin-specific keywords:
+
+        - scons-options:
+          (list of strings)
+          flags to pass to the build using the scons semantics for parameters.
+
+
+    $ snapcraft help tar-content
+    The plugin does not exist. Run `snapcraft list-plugins` to see the available plugins.
+
+
+    $ snapcraft help tar_content
+    The plugin has no documentation
+
+
+### tour
+
+    $ snapcraft tour
+    Snapcraft tour initialized in ./snapcraft-tour/
+    Instructions are in the README, or https://snapcraft.io/create/#begin
+
+Initialize the tour to an unexisting directory:
+
+    $ snapcraft tour idontexist
+    Snapcraft tour initialized in idontexist
+    Instructions are in the README, or https://snapcraft.io/create/#begin
+
+Initialize the tour to an existing directory:
+
+    $ snapcraft tour iexist
+    Snapcraft tour initialized in iexist
```

Comment:
This should be: Snapcraft tour initialized in iexist/snapcraft-tour/
https://bugs.launchpad.net/snapcraft/+bug/1604822